### DISCUSSION TITLE
Site templates/layouts based on PicoCSS

### DISCUSSION THREAD

**Discussion by richcorbs** [picocss/pico#618] [Ideas]
Would there be any value in having a few site templates/layouts (marketing site, dashboard, portfolio, etc) created using PicoCSS + minimal custom CSS?

I think there are at least two ways something like this would be valuable to the community:

- Having some site templates/layouts would accelerate someone getting started with PicoCSS.
- The implementation details of the site templates/layouts would be educational for anyone wanting to create a custom site using PicoCSS + minimal custom CSS.

PicoCSS is already super awesome and simple to get started but a couple of times I've wanted this or that and wished there was a pattern to follow: a hero section, a grid with uneven sized columns, etc.

Maybe we don't need site templates/layouts and just need additional examples of implementing custom elements like heroes or examples of how easy it is to add custom CSS and get what you want.

Anyway, just some ideas. Anyone else thinking the same thing?

**Comment by sweeperq** [comment]
Feel like this would be helpful for adoption. I just found out about Pico and like many of the default styles. Tried to set up a 'header header' 'leftnav body' 'footer footer' layout typically found in Admin templates and had some troubles. Ended up doing the "conditional" pico and applying the .pico class to my main body element and only using it there because it caused just as many layout problems as it fixed. A template with header, left nav (collapsible submenus), body, and footer would be great to have as a starting point.
